pkg/runtime/wfengine/component.go

```diff
@@ -107,7 +107,7 @@ func (c *workflowEngineComponent) Start(ctx context.Context, req *workflows.Star
 		return nil, fmt.Errorf("unable to start workflow: %w", err)
 	}
 
-	c.logger.Debugf("Created new workflow instance with ID '%s'", workflowID)
+	c.logger.Debugf("Created new workflow '%s' instance with ID '%s'", req.WorkflowName, workflowID)
```

Comment:
The other logs don't start capitalized, should this one?

Reply:
Normally logs start capitalized

Reply:
Capitalized every log to be consistent